File: QRCode/app/src/main/java/br/aplicacao/qrcode/model/Aluno.java
package br.aplicacao.qrcode.model;

public class Aluno {
   private int id_Aluno, rfid, matricula, numcel;
   private String nome;

   public Aluno(){

   }

   public void setNome(String nome){
        this.nome = nome;
   }

   public String getNome(){
        return nome;
   }

   public void setId_Aluno(int id_Aluno){
        this.id_Aluno = id_Aluno;
   }

   public int getId_Aluno(){
        return id_Aluno;
   }

   public void setRfid(int rfid){
        this.rfid = rfid;
   }

   public int getRfid(){
        return rfid;
   }

   public void setMatricula(int matricula){
        this.matricula = matricula;
   }

   public int getMatricula(){
        return matricula;
   }

   public void setNumCel(int numcel){
        this.numcel = numcel;
   }

   public int getNumCel(){
        return numcel;
   }


}

File: ios_android/lib/model/aluno.dart
//Read Only
class Aluno {
  int id = 0;
  int rfid = 0;
  int matricula = 0;
  String nome = '';
  int numcel = 0;

  Map toJson() => {
        "id_Aluno": id,
        "nome": nome,
        "rfid": rfid,
        "matricula": matricula,
        "numcel": numcel
      };

  Aluno.db(this.id, this.matricula, this.nome, this.numcel, this.rfid);
  Aluno();
  factory Aluno.fromJson(Map<String, dynamic> json) {
    return Aluno.db(
      json['id_Aluno'] as int,
      json['matricula'] as int,
      json['nome'] as String,
      json['numcel'] as int,
      json['rfid'] as int,
    );
  }
}
